src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/TagViewModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.UI.ViewModels.Models;
+using Windows.UI;
+
+namespace Microsoft.CmdPal.UI.ViewModels;
+
+public partial class TagViewModel(ITag _tag, TaskScheduler Scheduler) : ExtensionObjectViewModel
+{
+    private readonly ExtensionObject<ITag> _tagModel = new(_tag);
+
+    // Remember - "observable" properties from the model (via PropChanged)
+    // cannot be marked [ObservableProperty]
+    public string Text { get; private set; } = string.Empty;
+
+    public string Tooltip { get; private set; } = string.Empty;
+
+    public Color Color { get; private set; }
```

Comment:
Surprised to see we used `Windows.UI.Color` directly in the extension object, not sure how heavy that is to marshal across the WinRT boundary. @Sergio0694 probably knows. Typically, we'd have an alternate representation of it, and then convert it in the View layer.

Generally, we want to try and avoid any `Windows.` stuff in our models/viewmodels as it makes our code less portable in whatever far future might exist. It also then explicitly ties our testing to Windows, though with AOT and the need for C#/WinRT, we're kind of tied there anyway these days (which also means we can't avoid just having access to these in our ViewModel project anyway).

Reply:
You know what, in the Terminal we introduced our own WinRT Color struct too, for the exact same reason. You've got a good point, I'll go file a PR to merge that into the SDK instead

Reply:
fixed in #196